CIP-0067/README.md

```diff
@@ -0,0 +1,133 @@
+---
+CIP: 67
+Title: Asset Name Label Registry
+Authors: Alessandro Konrad <alessandro.konrad@live.de>, Thomas Vellekoop <thomas.vellekoop@iohk.io>
+Status: Draft
```

Comment:
```suggestion
Status: Proposed
```
@alessandrokonrad I think if we are going to merge this with your recently added Path to Active then it should be `Proposed` like CIP-0068 as well (cc @KtorZ)